Task2.2/MatrixExtension.cs

```diff
@@ -0,0 +1,27 @@
+﻿namespace Task2._2;
+
+public static class MatrixExtension
+{
+    public static Matrix AddMatrix(this Matrix a, Matrix b)
+    {
+        var elements = new int[Math.Max(a.Size, b.Size)];
+        var minSize = Math.Min(a.Size, b.Size);
+        var isASmaller = a.Size < b.Size;
```

Comment:
Better solution is to initialize elements using bigger matrix, and add elements from smaller using smaller matrix Size